<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="Tictactoe load test" timestamp="2017-12-15T16:13:04" hostname="localhost" time="2.053" errors="0" tests="1" skipped="0" disabled="0" failures="1">
  <testcase classname="Tictactoe load test" name="should be able to play 1000 games to end within timelimit of 1000ms" time="2.053">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (/home/daniel/HaustÃ¶nn 2017/HGOP/week3/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1973:23)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>